### ISSUE TITLE
Toggling "Large text" option in Android Accessibility should be respected by Flutter

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#5085]
Toggling "Large text" in Android's system accessibility menu should affect the size of text rendered by Flutter.

**Comment by eseidelGoogle** [comment]
There are a couple of similar bugs for iOS.  https://github.com/flutter/flutter/issues/4825 is one of them.

**Comment by abarth** [comment]
The framework mechanism for this will be done as https://github.com/flutter/flutter/issues/5873 .  This issue is able wiring that mechanism up to the platform-specific setting.

**Comment by abarth** [comment]
Specifically, this can be controlled by the MediaQuery.textScaleFactor.  We'll need to expose the textScaleFactor on Window, along side the devicePixelRatio.

**Comment by Hixie** [comment]
We do this now! @gspencergoog has more fixes coming for specific widgets, but I'm going to close this as done.
